### PR TITLE
Latestdoc

### PR DESCRIPTION
**Pull request by katunin-ng**
Before submitting a pull request, Please make sure the following:

## Description

Please explain the changes you made.

Link to design (if applicable)

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formating and linting run locally successfully
- [ ] All tests passing
- [x] UA review
- [ ] Design is documented
- [x ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,136 +3,29 @@
...
-#### Quick start example
-
-```go
-// Generate Makefile.mta file.
-mbt init 
+#### Multi-Target Application
 
-// Execute MTA project build.
-make -f Makefile.mta p=cf
+Before using this package, be sure you are familiar with the multi-target application concept and terminology. For background and detailed information, see the [Multi-Target Application Model](https://www.sap.com/documents/2016/06/e2f618e4-757c-0010-82c7-eda71af511fa.html) guide.
```
Comment (jhornik) [inline README.md]: Before using this package, make sure that you are familiar...

File: README.md [hunk 2]
```diff
@@ -3,136 +3,29 @@
...
-
- - [ ] Supports the extensibility framework.
- - [ ] Fully supports the advanced `mta.yaml` (3.1 > 3.2) schema.
-
-## Download and Installation
-
- There are two supported ways to install the multi-target application archive builder (MBT) tool:
+#### The Cloud MTA Build Tool overview
+The Cloud MTA Build Tool is a standalone command-line tool that builds a deployment-ready
+multi-target application (MTA) archive `.mtar` file from the artifacts of an MTA project according to the project’s MTA
```
Comment (jhornik) [inline README.md]: multitarget 

[no hyphen]

File: README.md [hunk 3]
```diff
@@ -3,136 +3,29 @@
...
+development descriptor (`mta.yaml` file) or from module build artifacts according to the MTA deployment descriptor (`mtad.yaml` file). Also it provides commands for runing intermediate build process steps, for example, the mta.yaml file validations,building a single module according to configurations in the development descriptor, generating deployment descriptor, etc.
```
Comment (jhornik) [inline README.md]: Also, it provides command for running intermediate build process steps; for example, the "mta.yaml" file validations, building a single module according to the configurations in the development descriptor, ....

File: README.md [hunk 4]
```diff
@@ -3,136 +3,29 @@
...
-        - Darwin / Linux
-          - Copy the binary file to the `~/usr/local/bin/` folder, for example: `cp mbt /usr/local/bin/`
-        - Windows
-          -  Copy the binary file `mbt.exe` to the `C:/Windows/` folder.
 
-**Use npm**:
-```
-npm install -g mbt
-```
+><b>For more information, see the the [Cloud MTA Build Tool user guide](docs/docs/index.md).</b>
```
Comment (jhornik) [inline README.md]: extra closing > at the beginning of the sentence.

File: docs/docs/configuration.md [hunk 5]
```diff
@@ -88,7 +88,9 @@ modules:
 <br>
 
 ####Configuring global build
-If you would like to run some builder process before running builders of the specific modules, define it in the `build-parameters` section at global level in mta.yaml as follows:
+
+If you would like to run some builder process before running builders of the specific modules, define it by using the `build-parameters` section at global level in mta.yaml as follows:
```
Comment (jhornik) [inline docs/docs/configuration.md]: If you would like to run a builder process before running the builders of the specific modules, define it by using the "build-parameters" section at the global level in the "mta.yaml" file as follows:

File: docs/docs/migration.md [hunk 6]
```diff
@@ -4,6 +4,8 @@ If you have previously used the [Multitarget Application Archive Builder](https:
 
 #### Features that handled differently in the Cloud MTA Build Tool
```
Comment (jhornik) [inline docs/docs/migration.md]: Features that are handled differently...

File: docs/docs/migration.md [hunk 7]
```diff
@@ -4,6 +4,8 @@ If you have previously used the [Multitarget Application Archive Builder](https:
...
 
+* For building an MTA project, the Cloud MTA build Tool uses `GNU Make` technology. Therefore, you should have `GNU Make` installed in your build environmnet. Please read the correponding sections for more information about [`GNU Make` installation](makefile.md) and [commands for building a project](usage.md#how-to-build-an-mta-archive-from-the-project-sources).
```
Comment (jhornik) [inline docs/docs/migration.md]: The Cloud MTA build Tool uses `GNU Make` technology for building MTA projects. Therefore, you should have `GNU Make` installed in your build environment. For more information, see the sections: [`GNU Make` installation](makefile.md) and [commands for building a project](usage.md#how-to-build-an-mta-archive-from-the-project-sources).

File: README.md (hunk 1 above)
Comment (katunin-ng) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (katunin-ng) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (katunin-ng) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (jhornik) [reply README.md]: Sorry I just noticed there's an extraneous "the" in the above sentence.

Comment (katunin-ng) [reply README.md]: '>' is used to make the whole line to be colored as a note to attract attention.

File: docs/docs/configuration.md (hunk 5 above)
Comment (katunin-ng) [reply docs/docs/configuration.md]: I will change it this way:
If you would like to run additional build steps before running the builders of the specific modules, define it by using the build-parameters section at the global level in the mta.yaml file as follows:

File: docs/docs/migration.md (hunk 6 above)
Comment (katunin-ng) [reply docs/docs/migration.md]: Updated

File: docs/docs/migration.md (hunk 7 above)
Comment (katunin-ng) [reply docs/docs/migration.md]: Updated

File: README.md (hunk 4 above)
Comment (katunin-ng) [reply README.md]: Double "the" fixed

Comment (jhornik) [reply README.md]: Thanks

File: README.md (hunk 1 above)
Comment (jhornik) [reply README.md]: Thanks

File: README.md (hunk 2 above)
Comment (jhornik) [reply README.md]: Thanks

File: README.md (hunk 3 above)
Comment (jhornik) [reply README.md]: Thanks

File: docs/docs/configuration.md (hunk 5 above)
Comment (jhornik) [reply docs/docs/configuration.md]: A minor correction below, alsot add back ticks:
...define them by using the `build-parameters` section at the global level in the `mta.yaml` file...

File: docs/docs/migration.md (hunk 6 above)
Comment (jhornik) [reply docs/docs/migration.md]: Thanks

File: docs/docs/migration.md (hunk 7 above)
Comment (jhornik) [reply docs/docs/migration.md]: Thanks

**Pull request merged**